### ISSUE TITLE
URL for docs

### ISSUE THREAD

**Issue by clburlison** [micromdm/docs#2]
In the #micromdm slack channel we decided to make the documentation it's own git repo 👍 

Next is we need to figure out the URL for the docs. The most common options I see are:
1. micromdm.io/docs
1. docs.micromdm.io

Since this site is currently hosted in a Google bucket either should be pretty easy to handle. I personally prefer option 1. This should also allow @groob to just dump this into the bucket under a /docs folder.

Once we decide we need to make sure and update the `baseURL` key in our `config.toml` to match.

**Comment by groob** [comment]
micromdm.io is deployed by serving a GCP storage bucket through the GCP loadbalancer/cdn
I would prefer `/docs` because it's an easy URL map in the loadbalancer config + 2 static buckets vs 2 loadbalancers with independent certificates. 
It also ends up being the cheaper option.

**Comment by joeblau** [comment]
Yeah I think choice 1 is better.  I remember reading something about preferring the `/docs` suffix over a custom subdomain.  That being said, i've also seen places that suggest using `www` subdomains over apex domains.
